screens/WelcomeScreen/WelcomeScreen.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View } from 'react-native';
+import { SocialIcon } from 'react-native-elements';
+import Colors from '../../constants/Colors';
+import { styles } from '../../components/styles';
+
+function WelcomeScreen(props) {
+  return (
+    <View style={styles.screen}>
+      <SocialIcon
+        title='Sign In With Buda'
+        button onPress={() => {
+          props.navigation.navigate({ routeName: 'Autentificacion' });
```

Comment:
Podemos poner las rutas en ingles para ser consistentes?
```suggestion
          props.navigation.navigate({ routeName: 'Authentication' });
```